src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/validation/FlowValidatorTest.java

```diff
@@ -256,4 +256,32 @@ public void checkForEncapsulationTypeRequirementCorrectTypeTest() throws Invalid
                 .build();
         flowValidator.checkForEncapsulationTypeRequirement(SRC_ENDPOINT, properties, TRANSIT_VLAN);
     }
+
+    private RequestedFlow getTestRequestWithMaxLatencyAndMaxLatencyTier2(Long maxLatency, Long maxLatencyTier2) {
+        return RequestedFlow.builder()
+                .flowId("firstFlow")
+                .maxLatency(maxLatency)
+                .maxLatencyTier2(maxLatencyTier2)
+                .srcSwitch(SWITCH_ID_1)
+                .srcPort(10)
+                .srcVlan(11)
+                .destSwitch(SWITCH_ID_2)
+                .destPort(12)
+                .destVlan(13)
+                .destVlan(13)
+                .detectConnectedDevices(new DetectConnectedDevices())
+                .build();
+    }
```

Comment:
minor comment for future: Usually private methods are placed at the end of a file.